tools/packaging/common/istio-iptables.sh

```diff
@@ -403,11 +403,8 @@ fi
 # 127.0.0.6 is bind connect from inbound passthrough cluster
 iptables -t nat -A ISTIO_OUTPUT -o lo -s 127.0.0.6/32 -j RETURN
 
-if [ -z "${DISABLE_REDIRECTION_ON_LOCAL_LOOPBACK-}" ]; then
-  # Redirect app calls back to itself via Envoy when using the service VIP or endpoint
-  # address, e.g. appN => Envoy (client) => Envoy (server) => appN.
-  iptables -t nat -A ISTIO_OUTPUT -o lo ! -d 127.0.0.1/32 -j ISTIO_IN_REDIRECT
-fi
+# Do not redirect app calls back to itself via Envoy when using the endpoint address
+iptables -t nat -A ISTIO_OUTPUT -o lo -j RETURN
```

Comment:
you need the ! -d 127... check here.

Reply:
in line 424, it has the filter ` iptables -t nat -A ISTIO_OUTPUT -d 127.0.0.1/32 -j RETURN`.

I think the CI failure is because the tls, for app talks to itself using vip. The outbout traffic destination addr is podip, and will not redirected to envoy inbound, thus no tls decrypt.

So i think `iptables -t nat -A ISTIO_OUTPUT -d 127.0.0.1/32 -j RETURN` add ! -m owner --uid-owner "${uid}" may work.

Reply:
But will try this as well